src/reconcile/rec_write.c

```diff
@@ -2967,7 +2974,11 @@ done:	if (F_ISSET(r, WT_REC_LOOKASIDE)) {
 		multi->page_las.unstable_txn = r->unstable_txn;
 		WT_ASSERT(session, r->unstable_txn != WT_TXN_NONE);
 		multi->page_las.max_timestamp = r->max_timestamp;
+		WT_ASSERT(session, r->unstable_durable_timestamp >=
+		    r->unstable_timestamp);
```

Comment:
You should split after `session,` - it's always preferred to keep logical expressions on a single line.

Reply:
As discussed, this assertion would fail too if all the updates were with prepare in-progress state. So introduced a boolean to track the prepare in-progress state for all the visited updates.